prosemirror-jdita-example/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM nginx:alpine
```

Comment:
Dockerfile looks good, but we need a bit more in there as these are likely to be published somewhere like Dockerhub... can you also add:

```
LABEL org.label-schema.build-date=${build-tstamp} \
      org.label-schema.description="ProseMirror JDita Example" \
      org.label-schema.name="prosemirror-jdita-example" \
      org.label-schema.schema-version="1.0" \
      org.label-schema.url="https://github.com/evolvedbinary/prosemirror-jdita" \
      org.label-schema.vcs-ref=${build-commit-abbrev} \
      org.label-schema.vcs-url="https://github.com/evolvedbinary/prosemirror-jdita" \
      org.label-schema.vendor="Evolved Binary"

COPY LICENSE /usr/share/nginx/html/LICENSE
```

I am not sure how you template in the `${value}` bits with Yarn/Node.js any ideas?

Reply:
We can run a script that updates Dockerfile after each build